ArduCopter/Log.cpp

```diff
@@ -451,6 +451,15 @@ void Copter::Log_Write_GuidedTarget(uint8_t target_type, const Vector3f& pos_tar
 // units and "Format characters" for field type information
 const struct LogStructure Copter::log_structure[] = {
     LOG_COMMON_STRUCTURES,
+    
+// @LoggerMessage: PTUN
+// @Description: Parameter Tuning information
```

Comment:
An @URL: or whatever it is to point to the channel 6 tuning guide would be good.  Specifically here because the `Param` entry references a specific enumeration of possible values - and with https://github.com/ArduPilot/ardupilot/pull/13858 in limbo that's going to be rather opaque just based on what's in here.